.github/workflows/tests.yml

```diff
@@ -109,6 +107,19 @@ jobs:
         with:
           name: coverage-jest
           path: .nyc_output/*.json
+  e2e:
+    needs: [prime-build]
+    name: Playwright tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/cached-install
+      - name: Build fabric.js
+        uses: ./.github/actions/build-fabric-cached
+      - name: Run playwright tests
+        run: |
+          npx playwright install
```

Comment:
doesn't npm ci install this too?

Reply:
seems not... at least before #9086 
I didn't check after

Reply:
I think it is needed for browsers etc

Reply:
https://playwright.dev/docs/ci-intro